docs/stack/client/GettingStarted.md

```diff
@@ -0,0 +1,41 @@
+## Installation
+
+Install the package with npm or yarn
+```sh
+npm install @daostack/client
+
+# Or
+
+yarn add @daostack/client
+```
+
+## Import
+
+Once installed, import the module in your project
+
+```js
+import { Arc } from '@daostack/client'
+
+//Or
+
+const client = require('@daostack/client')
+```
+
+## General Structure
```

Comment:
Adding a small note for referring to `How To Use` section.

Though I think this is kind of redundant because as part of mkdocs at the end of this page there is `Next` button that points to `How To Use` section. 🤷🏼‍♀️

Reply:
In technical writing is is always recommended to introduce at the beginning, summarize at the end and tell where you're going next.  Within reason, that redundancy is a good thing.

Here we are talking about "Getting Started" and I totally think part of getting started is to tell the user that the next step for them is to create and configure an instance of Arc, and make it easy by pointing to that section.  A "Next" button is less obvious and is not in the least informative in that respect